chord_metadata_service/chord/ingest.py

```diff
@@ -214,6 +216,29 @@
                 }
             ]
         },
+        WORKFLOW_READSET: {
+            "name": "Readset",
+            "description": "This workflow will copy readset files over to DRS.",
+            "data_type": DATA_TYPE_READSET,
+            "file": "readset.wdl",
+            "inputs": [
+                {
+                    "id": "readset_files",
+                    "type": "file[]",
+                    "required": True,
+                    "extensions": [".cram", ".bam", ".bigwig"]
```

Comment:
bigwigs are not read files typically I think

Reply:
I see. Basically, we need some name to represent all possible ExperimentResult file types. I will leave it for now and discuss with others what would be the best naming for it.